examples/routing/README.md

```diff
@@ -0,0 +1,61 @@
+# Routing
+With CMS React, you can use popular routing libraries like React-Router to enable SPA-style routing within your CMS website. The Pokedex example in this directory showcases how to set up SPA routing in your own project using React-Router. This documentation will walk through key aspects of our example, but for more information on React-Router, please visit their [docs](https://reactrouter.com/en/main).
```

Comment:
This all looks good, the only thing I think is missing is the usage of [:dynamic-slug] within the actual page to enable the SPA routing on the server.

Reply:
Addressed https://github.com/HubSpot/cms-react/pull/63/commits/299d776986722558d905484b85c7b6a51dc36026